starter-code/app.js

```diff
@@ -1,5 +1,58 @@
+'use strict';
+
 const express = require('express');
 const app = express();
-const Chuck  = require('chucknorris-io');
+//const expressLayouts = require('express-ejs-layouts');
+const Chuck = require('chucknorris-io');
 const client = new Chuck();
 
+//configure app
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+//app.use(expressLayouts);
+
+app.get('/', (req, res) => {
+  res.render('index');
+});
+
+app.get('/random', (req, res) => {
+  client
+    .getRandomJoke()
+    .then(response => {
+      res.send(`<p> ${response.value}</p>`);
```

Comment:
You want to render your view, not sending a p tag:
```
res.render('index', response.value); // Be careful, not the final code!
```
Furthermore, you send an object with render, so:
```
res.render('index', {joke: response.value});
```
Then you will acces from the view with:
```
<%= joke %>
```